Orochi/OrochiUtils.h

```diff
@@ -1,14 +1,22 @@
 #pragma once
 #include <Orochi/Orochi.h>
-#include <vector>
-#include <unordered_map>
-#include <string>
 #include <mutex>
+#include <string>
+#include <unordered_map>
+#include <vector>
 
-#if defined(_WIN32)
-	#define OROASSERT(x, y) if(!(x)) {__debugbreak();}
+#if defined( _WIN32 )
+#define OROASSERT( x, y )                                                                                                                                                                                                                                      \
+	if( !( x ) )                                                                                                                                                                                                                                               \
+	{                                                                                                                                                                                                                                                          \
+		__debugbreak();                                                                                                                                                                                                                                        \
+	}
 #else
-	#define OROASSERT(x, y) if(!(x)) {;}
+#define OROASSERT( x, y )                                                                                                                                                                                                                                      \
```

Comment:
Suggestion: On Linux, you can use th following construct:
#elif defined(__GNUC__)
#include <signal.h>
#define OROASSERT(cond) if (!(cond)) { raise(SIGTRAP); }
#else
#define OROASSERT(cond) 
#endif

Reply:
I rewrote the entire function